src/main/api/2/studio-api-2.yaml

```diff
@@ -1456,6 +1462,208 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'
 
+  /content:
+    get:
+      tags:
+        - content
+      summary: content
+      operationId: content
+      responses:
+        '200':
+          description: OK
+  /content/types:
+    get:
+      tags:
+        - content type
+      summary: Get content types
+      operationId: getContentTypes
+      parameters:
+        - name: siteId
+          in: query
+          description: Site ID
+          required: false
+          schema: 
+            type: string
+        - name: class
```

Comment:
Maybe `category` is a better name, instead of `class`? Just throwing it out there